dbcat/catalog/models.py

```diff
@@ -71,6 +71,8 @@ class CatSource(BaseModel):
     )
     region_name = Column(String)
     s3_staging_dir = Column(String)
+    mfa = Column(String)
+    aws_session_token = Column(String)
```

Comment:
Persisting these two values in the catalog DB doesn't really make much sense. Esp. the MFA token, since it can only ever be used a single time.